journal-systematic-palaeontology.csl

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Journal of Systematic Palaeontology</title>
+    <id>http://www.zotero.org/styles/journal-systematic-palaeontology</id>
+    <link href="http://www.zotero.org/styles/journal-systematic-palaeontology" rel="self"/>
+    <link href="http://www.zotero.org/styles/the-geological-society-of-london" rel="template"/>
+    <link href="http://www.tandf.co.uk/journals/authors/style/reference/ref_tjsp.pdf" rel="documentation"/>
+    <author>
+      <name>N.S. Vitek</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+	<issn>1477-2019</issn>
+    <eissn>1478-0941</eissn>
+    <summary>style for the Journal of Systematic Palaeontology</summary>
+    <updated>2016-06-17T01:36:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="symbol" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all" delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with=". " delimiter-precedes-last="never" delimiter=", ">
+        <name-part name="family" font-weight="bold"/>
+        <name-part name="given" font-weight="bold"/>
+      </name>
+      <et-al font-weight="bold"/>
+      <label form="short" prefix=" (" suffix=")"/>
```

Comment:
so the label shouldn't be bold? Again, not in the documentation -- is there an example in an article that you were able to find? Otherwise you could save yourself the trouble of the separate font-weight statements and just bold the whole macro.